src/utils/svg.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  select,
+  Selection,
+} from 'd3-selection';
+import { IMargin } from '../Histogram';
+
+export type TSelection = Selection<any, any, any, any>;
+export type TTextSelection = Selection<SVGTextElement, any, any, any>;
+
+interface ISizeProps {
+  margin: IMargin;
+  width: number | string;
+  height: number | string;
+  className: string;
+}
+
+export const makeSvg = (el: Element, svg: TSelection) => {
+  if (svg) {
+    svg.selectAll('svg > *').remove();
```

Comment:
I'm not an expert with d3, but wouldn't removing the root (just `svg.remove()`) remove all the children too?

Reply:
not sure - this has been working since day 0 so I think we can leave it like this

Reply:
:(